src/DataBase/EntryDatabase.cs

```diff
@@ -1,21 +1,25 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Linq.Expressions;
+using System.Threading.Tasks;
 using ePiggyWeb.DataBase.Models;
 using ePiggyWeb.DataManagement;
 using ePiggyWeb.DataManagement.Entries;
 using ePiggyWeb.Utilities;
+using Microsoft.EntityFrameworkCore;
 
 namespace ePiggyWeb.DataBase
 {
-    public static class EntryDatabase
+    public class EntryDatabase
     {
-        /*
```

Comment:
So in short this file was changed from static to use async/await?

Reply:
Yeah, in essence